advocacy_docs/partner_docs/CohesityBackupandRecover/04-ConfiguringCohesityBackupandRecoveryforPostgres.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Configuring Cohesity for Postgres'
+description: 'Walkthrough on configuring Cohesity Postgres Connector Agent'
```

Comment:
Should be "Walkthrough of configuring Cohesity" or whatever product name Lina gives us.